udata/auth/__init__.py

```diff
@@ -3,27 +3,28 @@
 
 import logging
 
-from flask import render_template, current_app, Blueprint, request, redirect
-from flask.ext.principal import (  # noqa
-    Permission as BasePermission, PermissionDenied, identity_loaded, RoleNeed,
-    UserNeed
-)
+from flask import current_app
+from flask import render_template
+
+from flask.ext.principal import identity_loaded
```

Comment:
Axel and I were chatting about this this afternoon. It's recommended by [the OpenStack style guidelines](https://pypi.python.org/pypi/hacking), and we think it's because it makes diffs easier to read, so we thought why not?